cve_bin_tool/checkers/samba.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for samba
+
+https://www.cvedetails.com/product/171/Samba-Samba.html?vendor_id=102
+
+"""
+from . import Checker
+
+
+class SambaChecker(Checker):
+    CONTAINS_PATTERNS = [r"SAMBA_([0-9]+\.[0-9]+\.[0-9]+)"]
+    FILENAME_PATTERNS = [
```

Comment:
https://www.samba.org/~ab/output/htmldocs/Samba3-HOWTO/compiling.html also lists "winbindd" and "nmbd" as necessary to configure in startup along with smbd. Are they part of the SMB package?

Reply:
These files do not have the strings which match the signatures in regex.

Reply:
Let's put a comment explaining that in the checker.  I don't know that it'll come up again, but I can't think of any reason not to be explicit about it.